sys-cluster/rdma-core/rdma-core-9999.ebuild

```diff
@@ -56,42 +56,34 @@ RDEPEND="${COMMON_DEPEND}
 
 BDEPEND="virtual/pkgconfig"
 
-PATCHES=( "${FILESDIR}"/optional_pandoc.patch )
-
 pkg_setup() {
 	use python && python-single-r1_pkg_setup
 
 }
 
 src_configure() {
 	local mycmakeargs=(
-		-DCMAKE_INSTALL_SYSCONFDIR=/etc
+		-DCMAKE_INSTALL_SYSCONFDIR="${EPREFIX}"/etc
 		-DCMAKE_INSTALL_RUNDIR=/run
 		-DCMAKE_INSTALL_SHAREDSTATEDIR=/var/lib
-		-DCMAKE_INSTALL_UDEV_RULESDIR="$(get_udevdir)"/rules.d
+		-DCMAKE_INSTALL_UDEV_RULESDIR="${EPREFIX}""$(get_udevdir)"/rules.d
 		-DCMAKE_INSTALL_SYSTEMD_SERVICEDIR="$(systemd_get_systemunitdir)"
-		-DCMAKE_DISABLE_FIND_PACKAGE_pandoc=yes
-		$(ver_test -ge 25 && echo -DCMAKE_DISABLE_FIND_PACKAGE_rst2man=yes)
 		-DCMAKE_DISABLE_FIND_PACKAGE_Systemd="$(usex systemd no yes)"
 		-DENABLE_VALGRIND="$(usex valgrind)"
 		-DENABLE_RESOLVE_NEIGH="$(usex neigh)"
 		-DENABLE_STATIC="$(usex static-libs)"
+		-DNO_PYVERBS="$(usex python OFF ON)"
```

Comment:
I’d consider doing usex !python but it’s up to you.

Reply:
Good idea, will change that.